scripts/run_integration_tests_live_migration.sh

```diff
@@ -45,6 +45,15 @@ if [ $? -ne 0 ]; then
 fi
 popd
 
+# Download Cloud Hypervisor binary from its last stable release
+LAST_RELEASE_VERSION="v22.0"
```

Comment:
The idea works, but that means we must recall to update this for every release.
So either you need to add some documentation to make sure we don't forget it, or you find a way to automate the retrieval of the value.

Reply:
I didn't find a good solution for automating this bit. Do you have any suggestions?